x-pack/plugins/observability_solution/entity_manager/server/lib/entities/ingest_pipeline/generate_history_processors.ts

```diff
@@ -132,6 +158,12 @@ export function generateHistoryProcessors(definition: EntityDefinition) {
         ignore_missing: true,
       },
     },
+    { script: { source: createIdentityFieldsPainlessScript(definition) } },
+    {
```

Comment:
can we keep the identityFields as a flat list of the fields from the definition?

i.e.
```
    {
      append: {
        field: 'entity.identityFields',
        value: definition.identityFields.map((x) => x.field),
      },
    },
```

Reply:
(and i think it helps make this clearer in the code if we make the temporary group_by destination `entity.identity`, so there is less dependency on correct ordering of processors in the ingest pipline)